docs/tasks/setup_rbac.md

```diff
@@ -0,0 +1,112 @@
+# Manage role-based access control
+
+This page shows you how to setup role-based acess control (RBAC) in your cluster
+to control the types of users that can view and create Kueue objects.
+
+The page is intended for a [batch administrator](/docs/tasks#batch-administrator).
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
+- [Kueue is installed](/README.md#installation).
+
+This page assumes you are already familiar with [RBAC in kubernetes](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+
+## ClusterRoles included in the installation
+
+When you install Kueue, a set of fine-grained ClusterRoles are created, to
+allow you to assign specific permissions to users. These ClusterRoles are the
+following:
```

Comment:
ok, I thought those were the roles created for the controllers; 

We don't need mention those at all, it is simpler and more to the point if we only present the coarse-grained roles related to the two personas we have.

Reply:
but if you insist on keeping it

```suggestion
When you install Kueue, the following set of fine-grained ClusterRoles are created to
allow you to assign specific permissions to users:
```

Reply:
Removed